test/resources/com/arpnetworking/metrics/portal/alerts/impl/PluggableAlertRepositoryTest.Alerts.json

```diff
@@ -1,173 +1,49 @@
 {
-  "version" : 0,
+  "version": 0,
   "alerts": [
     {
-      "name" : "FooIsTooHigh",
-      "description" : "Foo is too high.",
-      "enabled" : true,
-      "query" : {
-        "metrics": [
-          {
-            "tags": {
-              "host": ["host1"]
-            },
-            "name": "foo",
-            "limit": 10000,
-            "aggregators": [
-              {
-                "name": "sum",
-                "sampling": {
-                  "value": 1,
-                  "unit": "minutes"
-                }
-              },
-              {
-                "name": "filter",
-                "filter_op": "LT",
-                "threshold" : 10000.0
-              }
-            ]
-          }
-        ]
-      }
+      "name": "FooIsTooHigh",
+      "description": "Foo is too high.",
+      "enabled": true,
+      "query": "{\"metrics\": [{\"tags\": {\"host\": [\"host1\"]}, \"name\": \"foo\", \"limit\": 10000, \"aggregators\": [{\"name\": \"sum\", \"sampling\": {\"value\": 1, \"unit\": \"minutes\"}}, {\"name\": \"filter\", \"filter_op\": \"LT\", \"threshold\": 10000.0}]}]}"
```

Comment:
This was an automated change.

Every query field became a string containing the JSON that was originally in that field. The PluggableAlertRepository parsed as JSON since at least with the versioning it seemed OK to assume the structure and we get some nice early validation, but that doesn't play well with serializing from an internal model (since we can't guarantee it's JSON).

Since it's now a string, this will break any existing definition loading. I think this is acceptable because this is not production-ready yet and on our end we only have a single test definition which can easily be changed.